### DISCUSSION TITLE
Huggingface Diffusers library backend

### DISCUSSION THREAD

**Discussion by mchaker** [hlky/stable-diffusion-webui#701] [Feature suggestions]
Please please please use diffusers as the (or an optional) hlky webui backend.

It is FASTER than the default compvis repo.

HAS THE FOLLOWING FEATURES
* txt2img
* img2img
* inpainting
* **local pretrained weight usage (NOT dependent on the online API!)**

INFO: https://github.com/huggingface/diffusers/tree/main/src/diffusers/pipelines/stable_diffusion

must use [github version](https://github.com/huggingface/diffusers), not huggingface API version

**Comment by patrickvonplaten** [comment]
:wave: from the Diffusers team!

We would be more than happy to help you integrate `diffusers`. Let us know if they are any inconveniences. BTW, we'll make a release in 3 hours so with lots of new cool features.

**Reply by mchaker** [reply]
@hlky pinging for attention -- Diffusers team is here :)
